domains/andresarr.json

```
@@ -0,0 +1,9 @@
+{
+     "owner": {
+       "username": "Anonymous1223334444",
+       "email": "replit005@gmail.com"
+     },
+     "record": {
+       "CNAME": "https://portofoliowebdev.vercel.app"
```

`CNAME` record must contain domain name only.
```suggestion
       "CNAME": "portofoliowebdev.vercel.app"
```